src/core/ws/TextInputHandler.h

```diff
@@ -42,7 +42,7 @@
 
 #include <QObject>
 
-#include <dcWebservice/Handler.h>
+#include "dcWebservice/Handler.h"
```

Comment:
Those are really local includes. They get installed under a different path, namely <dc/webservice/...>. At some point we should improve the situation with namespaces in this project but right now it is the correct syntax.